Jenkinsfile

```diff
@@ -0,0 +1,80 @@
+String path = 'src/'
```

Comment:
some projects may require more then single source directory

Reply:
baby steps @partikus, baby steps.

Reply:
@jhryniuk it's a real value of CR that people leave comments and ideas for improvements. No one is hating people. Take it easy.